graph-builder/src/graph.rs

```diff
@@ -72,6 +77,12 @@ lazy_static! {
         "Total number of incoming HTTP client request to /v1/graph"
     )
     .unwrap();
+    static ref UP: Counter = Counter::with_opts(opts!(
+        "up",
```

Comment:
I have some concern that this name is misleading. `up` is a special metrics which is not controlled by the producer and has its well defined semantics: https://prometheus.io/docs/concepts/jobs_instances/

This is an info-metrics recording built-time details. The convention I've seen in other projects (e.g. Prometheus itself) is to name these `<APP>_build_info{...}`.

Reply:
Ah, good idea. Any other info except the commit hash we'd like to expose?

Reply:
Is there a way to get a staging/prod distinction into this metric?  I can't think of one, but having some kind of deployment-group-name would be nice.  Maybe Prometheus will layer that on when slurping.

Reply:
The staging-vs-prod information is already injected, even in a redundant way, by the infrastructure (they belong to different namespaces and datasources, and we are using the former for querying pruposes).

Reply:
The only info we can get there is being set during the build (see https://docs.rs/built/0.3.2/built/), I don't think we can distinguish stage and prod here as these are built from the single source

Reply:
@vrutkovs I had a quick look of what is exposed by `built`, and I didn't see anything else that would be immediately useful for our monitoring purposes.